### ISSUE TITLE
How to Render YCB Video dataset based custom data, if its textureless?

### ISSUE THREAD

**Issue by sag-Nik** [NVlabs/DeepIM-PyTorch#12]
FILE: ycb_video.py (in lib>datasets) 
LINE ~ 77,78

 # 3D model paths
self.model_mesh_paths = ['{}/{}/textured_simple.obj'.format(self._model_path, cls) for cls in self._classes_all]
self.model_texture_paths = ['{}/{}/texture_map.png'.format(self._model_path, cls) for cls in self._classes_all]

This is asking me to give texture files for the rendering purpose, but I am working on textureless data of YCB Video format generated by Isaac Sim.
Is there a work around?

**Comment by chengm0-0** [comment]
Hi, have you solved this question?

**Comment by sag-Nik** [comment]
Nope. It was a bit difficult for me to work with my custom object bcz the algorithm requires PoseCNN results to start with. So I altogether quit trying this framework.

You could try using a fake texture file, or generate a texture file for your object using Blender maybe. I was going to try those.
